source/documentation/sla/memory-balloon.html.md

```diff
@@ -55,28 +53,28 @@ oVirt's MOM sub-project integration[1](/wiki/Features/MomIntegration).
 
 *   VDSM already supports memory balloon as a device, and the following API should be used:
 
-      {'type': 'balloon ',
-      'device':'memballoon ',
-      'model': 'virtio' },     <--- 'none' may/should (need to check!) be used to disable the device.
+        {'type': 'balloon ',
+        'device':'memballoon ',
+        'model': 'virtio' },     <--- 'none' may/should be used to disable the device.
 
 ## Rest API
 
-*   A simple flag hould be added in the relevant actions.
+*   A simple flag should be added in the relevant actions.
```

Comment:
This is obsolete now. We should provide a links to:

- http://ovirt.github.io/ovirt-engine-api-model/4.1/#types/memory_policy
- http://ovirt.github.io/ovirt-engine-api-model/4.1/#types/cluster